drake/systems/plants/collision/ResultCollector.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DRAKE_SYSTEMS_PLANTS_COLLISION_RESULTCOLLECTOR_H_
+#define DRAKE_SYSTEMS_PLANTS_COLLISION_RESULTCOLLECTOR_H_
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+
+namespace DrakeCollision {
+Eigen::Vector3d toVector3d(const Eigen::Vector3d& vec);
+
+class ResultCollector {
+ public:
+  virtual ~ResultCollector() {}
+
+  virtual void addPointPairResult(const PointPair& result);
```

Comment:
Do you want me to open an issue? I don't presently have any expectation of continuing to work on collision-related things besides trying to get the FCL branch landed.